app/services/food_service.rb

```diff
@@ -0,0 +1,8 @@
+class FoodService
+  def self.foods_by_ingredient(ingredient)
+    search = "sweet potatoes"
+    response = Faraday.get("https://api.nal.usda.gov/fdc/v1/foods/search?api_key=#{ENV['FOOD_DB_KEY']}&query=#{search}&pageSize=10")
+
+    JSON.parse(response.body, symbolize_names: true)
+  end
+end
```

Comment:
To abstract out some of this functionality, I'd suggest making one method that's responsible for the faraday connection, and then within this method, calling to the endpoint you're trying to hit. You can also use a block to pass parameters to the url, it's a bit more conventional with ruby, and it's also cleaner. 

For example: 

```ruby 
 Faraday.new(url: 'https://api.nal.usda.gov') do |faraday|
      faraday.params['api_key'] = ENV['FOOD_DB_KEY']
    end
```